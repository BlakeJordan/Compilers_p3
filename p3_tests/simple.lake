bool glob;
int main(){
	int a;
	glob = a;
}
