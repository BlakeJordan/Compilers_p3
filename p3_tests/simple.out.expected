bool glob;
int main(){
	int a;
	glob(bool) = a(int);
}
